src/main/java/aquality/selenium/browser/devtools/EmulationTools.java

```diff
@@ -0,0 +1,34 @@
+package aquality.selenium.browser.devtools;
+
+import aquality.selenium.browser.AqualityServices;
+import aquality.selenium.core.localization.ILocalizedLogger;
+import org.openqa.selenium.devtools.DevTools;
+import org.openqa.selenium.devtools.v96.emulation.Emulation;
```

Comment:
heh, you use devtools v96 which are version-specific and could be not compatible with previous or future browser versions, how do you suggest to handle it?